src/infra/api/tags.ts

```diff
@@ -0,0 +1,9 @@
+import type { AxiosInstance } from "axios";
+
+export class PopularTagApi {
+  constructor(private api: AxiosInstance) {}
+
+  getPopularTag = () => {
+    return this.api.get(`http://localhost:3000/tags/popular`).then((response) => response.data);
```

Comment:
Suspense 오류가 여기때문에 그런것 같은데
api 객체에선 이미 baseurl이 설정되어 있으니까 pathname만 써도 돼!
```suggestion
    return this.api.get(`/tags/popular`).then((response) => response.data);
```
- 참고로 `localhost:3000`으로 하드코딩하면 storybook(`localhost:6006`)에선 돌아가지 않어!!

Reply:
여기서 로컬을 안붙이면 invalid url 이라고 또 오류가 나더라고..🤔 더 찾아봐야겠오..